boxsdk/auth/jwt_auth.py

```diff
@@ -44,6 +45,10 @@ def __init__(
             The ID of the Box Developer Edition enterprise.
         :type enterprise_id:
             `unicode`
+        :param jwt_key_id:
+            Key id for the jwt assertion
```

Comment:
To be consistent, can you:
- change id to ID
- change jwt to JWT
- end with a period